docs/source/tutorials/resnet/index.md

```diff
@@ -2,7 +2,7 @@
 
 # Classification with Torchvision's ResNet
 
-This tutorial demonstrates how to pretrain a ResNet model from [Torchvision](https://pytorch.org/vision/stable/) using LightlyTrain and then fine-tune it for classification using the PyTorch Lightning framework. We will perform both steps on the [Human Detection Dataset](https://www.kaggle.com/datasets/constantinwerner/human-detection-dataset/data) from Kaggle.
+This tutorial demonstrates how to pretrain a ResNet model from [Torchvision](https://pytorch.org/vision/stable/) using LightlyTrain and then fine-tune it for classification using [PyTorch Lightning](https://lightning.ai/docs/pytorch/stable/). We will perform both steps on two different human detection datasets from Kaggle. Such datasets can be used for training models to detect humans in images, a basic task in industries like security and autonomous driving.
```

Comment:
Maybe explain here that one dataset is used for pretraining and the other for fine-tuning. I would also mention that the pretraining one is unlabeled.